src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import isEqual from 'lodash/isEqual';
 import {ThemeContext} from '../../utilities/theme';
-import {Theme} from '../../utilities/theme/types';
-import {setColors} from '../../utilities/theme/utils';
+import {Theme, CSSProperties} from '../../utilities/theme/types';
+import {setColors} from '../../utilities/theme/legacy-utils';
```

Comment:
I'm name-spacing all the `topBar` theming concerns under legacy to optimize for deletion while preserving backwards compatibility.